example/speech_command/sc/dataset.py

```diff
@@ -10,9 +10,7 @@
     UserRawBuildExecutor,
 )
 
-dataset_dir = (
-    Path(__file__).parent.parent / "data" / "SpeechCommands" / "speech_commands_v0.02"
-)
+dataset_dir = Path(__file__).parent.parent / "data"
```

Comment:
`make train` will download the data into `data/SpeechCommands/speech_commands_v0.02` dir. If the users follow the example [readme](https://github.com/star-whale/starwhale/blob/main/docs/i18n/zh/docusaurus-plugin-content-docs/current/tutorials/speech.md#13-%E6%95%B0%E6%8D%AE%E5%87%86%E5%A4%87%E4%B8%8E%E6%A8%A1%E5%9E%8B%E8%AE%AD%E7%BB%83), the dataset build command is broken.

Reply:
done